docs/_src/tutorials/tutorials/1.md

```diff
@@ -71,8 +71,10 @@ You can start Elasticsearch on your local machine instance using Docker. If Dock
 
 
 ```python
-# Recommended: Start Elasticsearch using Docker
-#! docker run -d -p 9200:9200 -e "discovery.type=single-node" elasticsearch:7.9.2
+# Recommended: Start Elasticsearch using Docker via the Haystack utility function
+from haystack.utils import launch_es
+
+launch_es()
```

Comment:
That util function spawns a docker (which won't work on Colab). Do we keep an extra version in the notebooks that spawns elasticsearch directly on the host without docker?

Reply:
Yup in the notebooks, there is the option to use this utility function or to start from source